core/src/test/scala/unit/kafka/server/DescribeQuorumIntegrationTest.scala

```diff
@@ -26,16 +25,17 @@ import org.apache.kafka.common.protocol.{ApiKeys, Errors}
 import org.apache.kafka.common.requests.DescribeQuorumRequest.singletonRequest
 import org.apache.kafka.common.requests.{AbstractRequest, AbstractResponse, ApiVersionsRequest, ApiVersionsResponse, DescribeQuorumRequest, DescribeQuorumResponse}
 import org.junit.jupiter.api.Assertions._
-import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.{Tag, Timeout}
 import org.junit.jupiter.api.extension.ExtendWith
 
 import scala.jdk.CollectionConverters._
 import scala.reflect.ClassTag
 
+@Timeout(120)
 @ExtendWith(value = Array(classOf[ClusterTestExtensions]))
 @ClusterTestDefaults(clusterType = Type.KRAFT)
 @Tag("integration")
-class DescribeQuorumRequestTest(cluster: ClusterInstance) {
+class DescribeQuorumTest(cluster: ClusterInstance) {
```

Comment:
I'm a little confused what happened here, but it looks like we should get rid of this class and rename `DescribeQuorumTest` to `DescribeQuorumIntegrationTest`. Also, did we lose the admin integration test you added?

Reply:
I changed it so that the new test resides in `KRaftClusterTest`. This was Daivd's suggestion and I felt it made more sense there, especially because that new test used a different test framework than these ones.
The change in the name got included by mistake and I will revert it what it was.